### ISSUE TITLE
关于样本内数据（训练数据）的时间区间

### ISSUE THREAD

**Issue by pengxiao-song** [shiyu-coder/Kronos#40]
非常感谢作者的工作。

请问训练数据的时间区间是怎样的？

谢谢！

**Comment by shiyu-coder** [comment]
We are glad that our work has been helpful and inspiring to you.

The cut-off date for the training set is February 2024 (inclusive). The validation set spans from March to May 2024 (inclusive), and the out-of-sample period begins in June 2024.

Thank you.

**Comment by pengxiao-song** [comment]
Got it, thanks
